sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -16,6 +17,13 @@ export interface CloseConnectionOptions extends OperationOptions {
     reason?: string;
 }
 
+// @public
+export interface GenerateClientTokenOptions extends OperationOptions {
+    minutesToExpire?: number;
```

Comment:
`expirationTimeInMinutes`? I can't think of a good one, but the current name sounds a little strange to me

Reply:
FWIW I don't think `expirationTimeInMinutes` is better but I will change it since you do!